### PR TITLE
[FR] Difficulties

### PR DESCRIPTION
**Pull request by GTva**
French translation of osu! Wiki: Difficulties page.

### CODE REVIEW

File: wiki/Difficulties/fr.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+La difficulté d'une beatmap *décrit* le niveau de compétence requis pour pouvoir la finir.
+
+## Star rating (difficulté en étoiles)
+
+![Tranche de difficulté](img/SR-range.png "Star rating ranges")
+
+Pour les [Critères de ranking](/wiki/Ranking_Criteria), les tranches de niveau se divisent en plusieurs difficultés :
+
+- ![Icône easy](/wiki/shared/diff/easy-s.png "Easy icon") Easy: 0.0★–1.99★
```
Comment (RedNesto) [inline wiki/Difficulties/fr.md]:
```suggestion
- ![Icône easy](/wiki/shared/diff/easy-s.png "Icône easy") Easy: 0.0★–1.99★
```

File: wiki/Difficulties/fr.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+- ![Icône normal](/wiki/shared/diff/normal-s.png "Normal icon") Normal: 2.0★–2.69★
+- ![Icône hard](/wiki/shared/diff/hard-s.png "Hard icon") Hard: 2.7★–3.99★
+- ![Icône insane](/wiki/shared/diff/insane-s.png "Insane icon") Insane: 4.0★–5.29★
+- ![Icône expert](/wiki/shared/diff/expert-s.png "Expert icon") Expert: 5.3★–6.49★
+- ![Icône expert+](/wiki/shared/diff/expertplus-s.png "Expert+ icon") Expert+: >6.5★
+
+À noter que ces intervalles ne sont pas des limites; les maps peuvent significativement déviées de ces dernières et être cependant acceptées.
```
Comment (RedNesto) [inline wiki/Difficulties/fr.md]:
```suggestion
À noter que ces intervalles ne sont pas des limites; les maps qui ne les respectent pas peuvent toujours être acceptées.
```

File: wiki/Difficulties/fr.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+### Icônes
+
+Les tranches de difficulté ci-dessus déterminent quelle icône va être utilisée sur une beatmap.
+Suivre ces indicateurs peut vous donner une bonne estimation pour atteindre un niveau de difficulté équilibré. Vous pouvez trouver plusieurs tutoriels à suivre [ici](/wiki/Ranking_Criteria/#mapset) pour améliorer la difficulté de votre beatmapset.
+
+
```
Comment (RedNesto) [inline wiki/Difficulties/fr.md]: Ligne vide en trop

File: wiki/Difficulties/fr.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+## Niveau de difficulté
```
Comment (RedNesto) [inline wiki/Difficulties/fr.md]:
```suggestion
## Niveaux de difficulté
```

**Comment by fajar13k** [comment]
Let's also change the PR title to something like [FR] Difficulties instead (?)

**Pull request merged**